converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/DicomCastCoreResource.resx

```diff
@@ -120,6 +120,12 @@
   <data name="DataStoreOperationFailed" xml:space="preserve">
     <value>The data store operation failed.</value>
   </data>
+  <data name="FailedToValidateFhirVersion" xml:space="preserve">
+    <value>FHIR server version cannot be validated, should be R4</value>
+  </data>
+  <data name="FhirServerTransactionNotSupported" xml:space="preserve">
+    <value>FHIR server does not support transactions</value>
```

Comment:
nit: exceptions should provide details on how to fix it if possible. For this we could change it to.
"Configured FHIR service does not support transactions need to publish Fhir changes. Configure a FHIR service with transaction capability like FHIR-server"

Reply:
Updated, thanks!